docs/DeveloperGuide.md

```diff
@@ -350,42 +350,108 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* *`    | ~~tuition teacher~~ | ~~receive notifications when a student's performance drops below a certain threshold~~      | ~~proactively address any learning challenges they might be facing~~                                                 |
 | `*`      | ~~tuition teacher~~ | ~~be able to see a picture of them~~                                                        | ~~easily remember and identify them~~                                                                                |
 
+***
 
 ### Use cases
 
-(For all use cases below, the **System** is the `EduManage` and the **Actor** is the `tuition teacher`, unless specified otherwise)
+(For all use cases below, the **System** is `EduManage` and the **Actor** is the `tuition teacher`, unless specified otherwise)
+
+***
+
+**Use case: Add a student**
+
+**MSS**
+
+1. Tuition teacher requests to add a student by inputting the student's name, phone number, emergency contact, address and other optional fields if needed.
```

Comment:
no need if needed since optional already implies that they can be included or omitted.